opentracing-spring-cloud/src/main/java/io/opentracing/contrib/spring/cloud/feign/FeignTracingAutoConfiguration.java

```diff
@@ -19,10 +19,12 @@
 
 /**
  * @author Pavol Loffay
+ * @author Eddú Meléndez
  */
 @Configuration
 @ConditionalOnClass(Client.class)
 @AutoConfigureBefore(FeignAutoConfiguration.class)
+@ConditionalOnProperty(prefix = "opentracing.spring.cloud.feign", name = "enabled", havingValue = "true", matchIfMissing = true)
```

Comment:
We have only one property so prefix/name can be omitted  `value = opentracing.spring.cloud.feign.enabled`. Default values are defined in `additional-spring-configuration-metadata.json` can it be omitted too?

Reply:
default value in `additional-spring-configuration-metadata.json` it's not actually the real default value. It is just for information purpose. Hints when we are adding new properties in `application.properties`